Umbraco-Cloud/Upgrades/Moving-From-Courier-To-Deploy/index.md

```diff
@@ -10,47 +10,59 @@ Courier has been replaced by Deploy, and Deploy is highly specialized to be work
 In normal circumstances, the move from Courier to Deploy will happen automatically when you click the button in the portal for upgrading to Umbraco 7.6. 
 If for some reason your Umbraco project was upgraded, but not switched to Deploy. i.e. if you did the upgrade manually.
 
-This is a guide for moving from Courier to Deploy. First step is to acquire the latest Umbraco Deploy package and a valid Deploy Key from the support heroes. Use the in-app chat in the Umbraco Cloud portal to get this.
+This is a guide for moving from Courier to Deploy. 
+The first step is to acquire the latest Umbraco Deploy package from [http://nightly.umbraco.org/?container=umbraco-deploy-release](http://nightly.umbraco.org/?container=umbraco-deploy-release)
+
+For a version 7.x website, choose the latest "UmbracoDeploy.v2.x.x.zip" file.
+
+Next, request a valid Deploy Key from the support heroes. Use the in-app chat in the Umbraco Cloud portal to get this.
```

Comment:
```suggestion
Next, request a valid Deploy Key from Umbraco Support. This can be done through the in-app chat in the Umbraco Cloud Portal or by reaching out to contact@umbraco.com.
```

Removed "support heroes", as this might become a bit internal.
Also provided one more option for getting in touch with our support.

Reply:
Great improvement :-)